### PR TITLE
lvm: Add emulated_image_size for EmulatedLVM

### PR DESCRIPTION
**Pull request by zhencliu**
Signed-off-by: Zhenchao Liu <zhencliu@redhat.com>

ID: 1904389

### CODE REVIEW

**Comment by zhencliu** [comment]
hi @YongxueHong , can you please help to review this patch? Please refer to the mail sent to you for the details, thanks much.

**Comment by zhencliu** [comment]
hi @YongxueHong , change done based our talk this afternoon, please help to make a double check when you are available, thanks.

File: virttest/lvm.py [hunk 1]
```diff
@@ -634,7 +634,7 @@ def make_emulate_image(self):
         """
         Create emulate image via dd with 8M block size;
         """
-        img_size = self.params["lv_size"]
+        img_size = normalize_data_size(self.params["lv_size"])
```
Comment (YongxueHong) [inline virttest/lvm.py:637]: Sorry, I forget that `params["lv_size"]` has been `normalize_data_size` in `__format_params`.
Please refer:
```
class LVM(object):
...
   def __init__(self, params):
        self.params = self.__format_params(params)
        ...
    def __format_params(self, params):
        lv_size = params.get("lv_size")
        if lv_size is None:
            lv_size = params["image_size"]
        params["lv_size"] = normalize_data_size(lv_size)
```
so I think we don't need to `normalize_data_size` again.

Comment (zhencliu) [reply virttest/lvm.py]: updated

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**